internal/internal_public.go

```diff
@@ -33,6 +33,8 @@ import (
 	s "go.uber.org/cadence/.gen/go/shared"
 )
 
+//go:generate mockery --srcpkg . --name WorkflowTaskHandler --output . --outpkg internal --inpackage --with-expecter --case snake --boilerplate-file  ../LICENSE
```

Comment:
```suggestion
//go:generate mockery --name WorkflowTaskHandler --inpackage --with-expecter --case snake --boilerplate-file  ../LICENSE
```
quite a bit simpler and produces identical output (since those three flags are exactly what `--inpackage` does)